src/bodse/timetable/GTFS_download.py

```diff
@@ -0,0 +1,107 @@
+import requests
+from datetime import datetime, timedelta
+from pathlib import Path
+import zipfile
+import os
+from clint.textui import progress
+import sys
+
+
+def download_gtfs_feed(directory):
+    current_datetime = datetime.now()
+    current_date_time = current_datetime.strftime("%Y%m%d_%H-%M-%S")
+
+    try:
+        print("Downloading started")
```

Comment:
Switch print statements to logging statements e.g. `LOG.info()`, see other modules for examples
```suggestion
        LOG.info("Downloading started")
```